app/imports/server/discord/sendWebhook.js

```diff
@@ -1,20 +1,35 @@
 import Discord from 'discord.js'
 export default function sendWebhook({ webhookURL, data = {} }) {
-  //webhookURL = https://discordapp.com/api/webhooks/<id>/<token>
-  let urlArray = webhookURL.split('/');
-  let token = urlArray.pop();
-  let id = urlArray.pop();
 
-  // prevent discord mention exploit
-  data.disableMentions = 'all';
+  // check if this is a _discord_ webhook
+  if (!webhookURL || !webhookURL.includes('discordapp.com/api/webhooks')) {
```

Comment:
looks like a logic bug, should this really be sending discord webhooks if the URL does not contain `discordapp.com...`?